### PR TITLE
PointerIndex Error

### PR DESCRIPTION
**Pull request by ppamorim**
I've tried to solve pointerIndex error

### CODE REVIEW

**Comment by pedrovgs** [comment]
Is this pull request a fix for #47?

**Comment by ppamorim** [comment]
I think it's. Becase another library have the same issue and I've do the same for to fix it...

File: draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java [hunk 1]
```diff
@@ -323,14 +326,24 @@ public boolean isClosed() {
    * @param ev captured.
    * @return true if the view is going to process the touch event or false if not.
    */
-  @Override public boolean onInterceptTouchEvent(MotionEvent ev) {
+  @Override public boolean onInterceptTouchEvent(@NonNull MotionEvent ev) {
```
Comment (pedrovgs) [inline draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java]: This annotation has no sense. You are overriding this method from the Android SDK and is the Android SDK who is going to call this method. The contract specified in the onInterceptTouchEvent declaration says MotinEvent can't be null. Please, remove this annotation and the import.

File: draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java [hunk 2]
```diff
@@ -52,7 +53,9 @@
   private static final int ONE_HUNDRED = 100;
   private static final float SENSITIVITY = 1f;
   private static final boolean DEFAULT_TOP_VIEW_RESIZE = false;
+  private static final int INVALID_POINTER = -1;
 
+  private int mActivePointerId = INVALID_POINTER;
```
Comment (pedrovgs) [inline draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java]: This repository doesn't use "m" notation. Please rename this variable to activePointerId.

File: draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java [hunk 3]
```diff
@@ -342,7 +355,14 @@ public boolean isClosed() {
    * @param ev captured.
    * @return true if the touch event is realized over the drag or second view.
    */
-  @Override public boolean onTouchEvent(MotionEvent ev) {
+  @Override public boolean onTouchEvent(@NonNull MotionEvent ev) {
```
Comment (pedrovgs) [inline draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java]: Remember to remove this annotation.

File: draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java (hunk 2 above)
Comment (ppamorim) [inline draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java]: My fault. Bad day...

File: draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java (hunk 1 above)
Comment (ppamorim) [inline draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java]: Bad AS...

**Comment by pedrovgs** [comment]
Thanks for your pull request!

**Pull request merged**